scripts/pyspelling/custom_dictionary.txt

```diff
@@ -0,0 +1,6 @@
+Splink
+PyPI
+DuckDB
+deduplication
+Conda
+Quickstart
```

Comment:
Are these words case sensitive? I imagine not

Reply:
Yes they are case sensitive.

In the case of 'Splink' this is useful for picking up lowercase versions in the text, but this does mean that correct `splink`s in code lines will also be flagged as errors. Something that can be addressed in a follow-up PR me thinks